tests/test_builder_containerbuild.py

```diff
@@ -444,7 +444,7 @@ def _mock_osbs(self, koji_build_id, src, koji_task_id, orchestrator=False, sourc
         (osbs
             .should_receive('wait_for_build_to_get_scheduled')
             .with_args('os-build-id'))
-        (osbs.should_receive('cancel_build').never)  # pylint: disable=expression-not-assigned
+        (osbs.should_receive('cancel_build').never())  # pylint: disable=expression-not-assigned
```

Comment:
Please remove pylint comment, it should be fixed by your change

Reply:
done